CompetitionTemplate/src/competition/subsystems/turret/rotation/commands/RotateTurretWithGamepadCommand.java

```diff
@@ -25,6 +29,13 @@ public void initialize() {
 
     @Override
     public void execute() {
-        rotationSubsystem.setRotationPower(oi.leftJoystick.getButton(8) ? oi.leftJoystick.getRawAxis(4) : 0);
+        double leftIntent = leftTrigger.getVector().y;
+        double rightIntent = -1 * rightTrigger.getVector().y;
+        
+        if(rightIntent != 0 && leftIntent != 0){
```

Comment:
This feels weird. Firstly, if either of the intents are nonzero, it currently _zeros_ the output: isn't that reversed?

Second, logic like this will check if either is depressed, but if both of them are the same value the final intent value (the sum) will be zero and nothing will happen anyway. So, not only does the `if` not actually do anything, but it doesn't check all cases where the intent is zero.

Maybe instead of comparing with zero like you're currently doing, you want to threshold it so that anything below some value is snapped to zero.